tests/visual/scheduler.html

```diff
@@ -639,6 +641,181 @@
             </div>
         </div>
 
+        <div id="scheduler" style="height: 600px;" data-role="scheduler" class="k-widget k-scheduler k-floatwrap" role="grid" aria-multiselectable="true">
```

Comment:
You can clear the `role` and `aria` labels and attributes from the test markup, if you are not testing them

Reply:
Provide info as a comment that this is an Angular scheduler test, please